Bundle/CoreBundle/doc/ManageViews.md

```diff
@@ -0,0 +1,31 @@
+#Manage Views
+
+##Chain
+
+A chain list all views managers and allow to find.
+
+##Managers
+
+###ViewManagerInterface
+
+This interface describe all methods that we need for a view.
+For the moment we just a method to build the correct ViewReference associated.
+
+###BaseViewManager
+
+This Class have the setter for basic views Managers service dependence.
+
+###ViewsManager
+
+Managers must manage all events associated to a view.
+Extends baseViewManager and implements viewManagerInterface
```

Comment:
Please pay attention to the case, the first letter of a class must be capitalized.